src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -592,11 +592,19 @@ private void Resize(int newSize, bool forceNewHashCodes)
 
         public bool Remove(TKey key)
         {
+            TValue unused;
+            return Remove(key, out unused);
```

Comment:
That doesn't seem to compile in our compiler right now. If it did, I think it's fine.